lisa/tools/reboot.py

```diff
@@ -27,6 +29,23 @@ def command(self) -> str:
     def _check_exists(self) -> bool:
         return True
 
+    def reboot_and_check_panic(self, log_path: Path) -> None:
+        if self.node.features._is_supported(SerialConsole):
+            try:
+                self.reboot()
+            except Exception as identifier:
+                # if there is any panic, fail before partial pass
+                serial_console = self.node.features.__getitem__(SerialConsole)
```

Comment:
There should be soft check, or it may be failed, if current node has no SerialConsole feature.

Reply:
It's inside the if condition - if self.node.features._is_supported(SerialConsole):
Shouldn't it work always?